source/opt/ir_context.h

```diff
@@ -490,6 +494,16 @@ class IRContext {
 
   uint32_t max_id_bound() const { return max_id_bound_; }
   void set_max_id_bound(uint32_t new_bound) { max_id_bound_ = new_bound; }
+    
+  bool preserve_bindings() const { return preserve_bindings_; }
+  void set_preserve_bindings(bool preserve_bindings) {
```

Comment:
GCC complains because `preserve_bindings` is the name of this parameter and the function above.  Can you fix that?